.gitignore

```diff
@@ -9,3 +9,9 @@
 node_modules/
 vendor/
 .terraform/
+
+# tfc-dump creates .json files that don't belong in version control
+*.json
+*.env
```

Comment:
It may be worth adding `*.env` to the ".dockerignore" file as well, to protect against including a local.env file if you ever build this locally and push it.